free-2.md

```diff
@@ -1,44 +1,85 @@
 # Free project 2
 
+MarkTeX?? : Simple and possible to use in a WYSIWYG editor like Markdown,
+but targeted at LaTeX's domain.
```

Comment:
Markdown isn't inherently WYSIWYG, but it is simple enough that good WYSIWYG editors exist. LaTeX has a few like LyX but they aren't easy to make and have issues. That's what I meant.